apps/system/style/zindex.css

```diff
@@ -334,3 +335,14 @@
 #screen > [data-z-index-level="accessibility-captions"] {
   z-index: 2147483647;
 }
+
+/**
+ * Global overlays are on top of everything, all the time.
+ */
+#screen > [data-z-index-level="global-overlays"] {
+  z-index: 2147483647;
+}
+
+#screen > [data-z-index-level="global-overlays"] > .appWindow.overlayWindow {
+  z-index: 2147483647;
```

Comment:
This is also a good point. However we need this window to be on top of *any* window. It is rendered transparent and allowing to pass the pointer events, so unless the overlay window renders some UI that captures the pointer events, this should never happen. I agree that there's still a small chance that this happens though.